src/components/modal/ModalFooter.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export interface IModalFooterProps {
```

Comment:
Comment from lbergeron in Dynamics repo: "Tu ne devrais pas supporter d'avoir des classes CSS supplémentaires comme pour les autres components?"